cudax/include/cuda/experimental/__cuco/detail/hash_functions/murmurhash3.cuh

```diff
@@ -0,0 +1,831 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of CUDA Experimental in CUDA C++ Core Libraries,
+// under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES.
+//
+//===----------------------------------------------------------------------===//
+
+/* MurmurHash3_32 implementation from
+ * https://github.com/aappleby/smhasher/blob/master/src/MurmurHash3.cpp
+ * -----------------------------------------------------------------------------
+ * MurmurHash3 was written by Austin Appleby, and is placed in the public domain. The author
+ * hereby disclaims copyright to this source code.
+ *
+ * Note - The x86 and x64 versions do _not_ produce the same results, as the algorithms are
+ * optimized for their respective platforms. You can still compile and run any of them on any
+ * platform, but your performance with the non-native version will be less than optimal.
+ */
+
+#ifndef _CUDAX__CUCO_DETAIL_HASH_FUNCTIONS_MURMURHASH3_CUH
+#define _CUDAX__CUCO_DETAIL_HASH_FUNCTIONS_MURMURHASH3_CUH
+
+#include <cuda/__cccl_config>
+
+#if defined(_CCCL_IMPLICIT_SYSTEM_HEADER_GCC)
+#  pragma GCC system_header
+#elif defined(_CCCL_IMPLICIT_SYSTEM_HEADER_CLANG)
+#  pragma clang system_header
+#elif defined(_CCCL_IMPLICIT_SYSTEM_HEADER_MSVC)
+#  pragma system_header
+#endif // no system header
+
+#include <cuda/__utility/static_for.h>
+#include <cuda/std/__bit/bit_cast.h>
+#include <cuda/std/__bit/rotate.h>
+#include <cuda/std/array>
+#include <cuda/std/cstddef>
+#include <cuda/std/cstdint>
+#include <cuda/std/span>
+
+#include <cuda/experimental/__cuco/detail/hash_functions/utils.cuh>
+
+#include <cuda/std/__cccl/prologue.h>
+
+namespace cuda::experimental::cuco::__detail
+{
+
+template <typename _Key>
+[[nodiscard]] _CCCL_API constexpr _CUDA_VSTD::uint32_t __fmix32(_Key __key, _CUDA_VSTD::uint32_t __seed = 0) noexcept
+{
+  static_assert(sizeof(_Key) == 4, "Key type must be 4 bytes in size.");
+
+  _CUDA_VSTD::uint32_t __h = _CUDA_VSTD::bit_cast<_CUDA_VSTD::uint32_t>(__key) ^ __seed;
```

Comment:
Sorry to say that but we dropped the use of those macros recently, so you should replace all `_CUDA_VSTD` with `::cuda::std`